.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  extends: ["react-app", "plugin:jsx-a11y/recommended", "prettier"],
+  plugins: ["jsx-a11y"],
```

Comment:
```suggestion
  extends: ["react-app", "prettier"],
  plugins: ["jsx-a11y"],
```
According to [this](https://www.npmjs.com/package/eslint-plugin-jsx-a11y) we don't need to do both `extends` and `plugin`. Is there a reason both are there that I was unable to locate?

![image](https://github.com/codefordallas/codefordallas.github.io/assets/20706498/f6e98da2-abac-45ce-b7a6-06c86783cbb1)

I removed from extension so users could see both variables.